code/modules/medical/cloner_defects.dm

```diff
@@ -450,24 +477,61 @@ ABSTRACT_TYPE(/datum/cloner_defect/organ_damage)
 
 	on_add()
 		. = ..()
-		if (data["trait_id"])
+		if (data["trait_id"] && !src.owner.traitHolder?.hasTrait(data["trait_id"]))
 			src.owner.traitHolder.addTrait(data["trait_id"])
-		else
+		else if (src.owner.bioHolder.HasEffect(data["bioeffect_id"]))
 			src.owner.bioHolder.AddEffect(data["bioeffect_id"])
 			var/datum/bioEffect/effect = src.owner.bioHolder.GetEffect(data["bioeffect_id"]) // this suuuucks
 			effect.curable_by_mutadone = FALSE
 
+/datum/cloner_defect/biotrait/plasmalungs
+	name = "Plasma Lung Infection"
+	desc = "Subject's lungs have become infected and are now only capable of breathing plasma."
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
+	weight = 50
+	effect_type_pool = list(/datum/trait/plasmalungs)
+
+/datum/cloner_defect/biotrait/blindness
+	name = "Permanent Retina Damage"
+	desc = "Subject's eyes have become damaged and the subject is now blind."
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
+	effect_type_pool = list(/datum/bioEffect/blind)
+
+/datum/cloner_defect/biotrait/deafness
+	name = "Permanent Cochlea Damage"
+	desc = "Subject's ears have become damaged and the subject is now deaf."
+	severity = CLONER_DEFECT_SEVERITY_MAJOR
+	effect_type_pool = list(/datum/bioEffect/deaf)
+
+/datum/cloner_defect/biotrait/sleepy
```

Comment:
Cloning defects should ideally be something fairly obvious that can be fixed or mitigated by the average MD right after you spill out of the tube, being randomly stunned for the rest of the round is not that and will just make people angry.